File: m2/Personaje.cs
public class Personaje
{
    // Propiedades
    public string Nombre { get; set; }
    public int Defensa { get; set; }
    public int Poder { get; set; }
    public int Salud { get; set; }

    // Constructor vacío
    public Personaje()
    {
    }

    // Constructor con parámetros
    public Personaje(string nombre, int defensa, int poder, int salud)
    {
        Nombre = nombre;
        Defensa = defensa;
        Poder = poder;
        Salud = salud;
    }

    // Método para mostrar información del personaje
    public override string ToString()
    {
        return $"Personaje {{ Nombre = {Nombre}, Defensa = {Defensa}, Poder = {Poder}, Salud = {Salud} }}";
    }
}

File: m2/Personaje.java
package m2;
public class Personaje {
    private String nombre;
    private int defensa;
    private int poder;
    private int salud;

    public Personaje(String nombre, int defensa, int poder, int salud) {
        this.nombre = nombre;
        this.defensa = defensa;
        this.poder = poder;
        this.salud = salud;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getDefensa() {
        return defensa;
    }

    public void setDefensa(int defensa) {
        this.defensa = defensa;
    }

    public int getPoder() {
        return poder;
    }

    public void setPoder(int poder) {
        this.poder = poder;
    }

    public int getSalud() {
        return salud;
    }

    public void setSalud(int salud) {
        this.salud = salud;
    }
}
